pom.xml

```diff
@@ -44,6 +44,7 @@
     <coveralls.maven.plugin.version>4.3.0</coveralls.maven.plugin.version>
     <failsafe.maven.plugin.version>2.22.0</failsafe.maven.plugin.version>
     <findbugs.maven.plugin.version>3.0.5</findbugs.maven.plugin.version>
+    <spotbugs.maven.plugin.version>3.1.12.2</spotbugs.maven.plugin.version>
     <hotels.oss.plugin.config.version>1.2.2</hotels.oss.plugin.config.version>
```

Comment:
This needs to become 1.3.0 which I've just released. Not sure how this was working for you locally with 1.2.2?

Reply:
Oh yeah i tested with the 1.3.0-snapshot but wasn't sure if i should update versions yet but ill fix that now :)

Reply:
Regarding the first comment^: https://maven.apache.org/plugins/maven-dependency-plugin/usage.html#Overwrite_Rules
It looks like setting it to true just means that it force overwrites any files found in the destination every time you unpack. It could be worth changing it to true if you want to make sure you're unpacking everything thats available each time?

Reply:
Yes, let's do that. Add both changes to the CHANGELOG.

Reply:
done :)